ddtrace/tracer/option.go

```diff
@@ -264,6 +264,9 @@ type config struct {
 	// traceSampleRules holds the trace sampling rules
 	traceSampleRules dynamicConfig[[]SamplingRule]
 
+	// spanSampleRules holds the trace sampling rules
+	spanSampleRules dynamicConfig[[]SamplingRule]
+
```

Comment:
We do not support dynamic configuration of span rules at the moment. It is OK if you want to add this here to make it easier for future support. In that case, please add a comment to state as such.

Reply:
To avoid confusion, I'll just remove this.